gaudi/README.md

```diff
@@ -0,0 +1,84 @@
+# Setting up HabanaAI Operator
+
+## Overview
+The steps mentioned in this document to deploy and configure [Habana AI Operator](https://catalog.redhat.com/software/container-stacks/detail/64342b3bcbfbb9a6588ce8dd) on Red Hat OpenShift are consolidated and referenced from [HabanaAI Operator for OpenShift](https://docs.habana.ai/en/latest/Orchestration/HabanaAI_Operator/index.html).
+
+## Prerequisities
+- Provisioned RHOCP cluster. Follow steps [here](/README.md).
+
+## Setting up OpenShift Environment
+- Setup Machine Configuration to update Firmware search path using below command. Please note that this step will reboot the nodes.
+```
+oc apply -f https://raw.githubusercontent.com/intel/intel-technology-enabling-for-openshift/main/gaudi/gaudi_firmware_path.yaml
+```
+ 
+- Setup Node Feature Discovery (NFD) instance using below command:
+```
+oc apply -f https://raw.githubusercontent.com/intel/intel-technology-enabling-for-openshift/main/gaudi/gaudi_nfd_instance_openshift.yaml
+```
+Verify NFD has labelled the node correctly:
+```
+oc get no -o json | jq '.items[].metadata.labels' | grep pci-1da3
+
+"feature.node.kubernetes.io/pci-1da3.present": "true",
+```
+NFD discovers underlying Habana Gaudi using its PCI device class and the vendor. Here `"feature.node.kubernetes.io/pci-1da3.present": "true"`, represents node has PCI card from the vendor 1da3 which is HabanaLabs. 
+
+## Install HabanaAI Operator on Red Hat OpenShift
+### Installation via web console
+Follow the steps below to install Intel Device Plugins Operator using OpenShift web console:
```

Comment:
Intel Device Plagins Operator -> Habana AI Operator

Reply:
Made the requested changes.